examples/go/with_proto/go/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"github.com/bazelbuild/intellij/examples/go/with_proto/go/lib"
+	"github.com/bazelbuild/intellij/examples/go/with_proto/proto"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	num := 3
+	fmt.Printf("AddToTwo(%s) = %s", num, lib.AddToTwo(num))
+}
+
+func serv(grpcSrv *grpc.Server, protoSrv *proto.FooServiceServer) {
```

Comment:
Can you clarify, maybe with a comment, how we can test this function? i.e. what should we check that is working as expected?